resources/lang/vi/validation.php

```diff
@@ -0,0 +1,195 @@
+    'prohibited' => 'Trường :attribute bị cấm.',
+    'prohibited_if' => 'Trường :attribute bị cấm khi :other là :value.',
+    'prohibited_unless' => 'Trường :attribute bị cấm trừ khi :other là một trong :values.',
+    'prohibits' => 'The :attribute field prohibits :other from being present.',
+    'regex' => 'Trường :attribute có định dạng không hợp lệ.',
+    'required' => 'Trường :attribute không được bỏ trống.',
+    'required_if' => 'Trường :attribute không được bỏ trống khi trường :other là :value.',
+    'required_unless' => 'Trường :attribute không được bỏ trống trừ khi :other là :values.',
+    'required_with' => 'Trường :attribute không được bỏ trống khi một trong :values có giá trị.',
+    'required_with_all' => 'Trường :attribute không được bỏ trống khi tất cả :values có giá trị.',
+    'required_without' => 'Trường :attribute không được bỏ trống khi một trong :values không có giá trị.',
+    'required_without_all' => 'Trường :attribute không được bỏ trống khi tất cả :values không có giá trị.',
+    'same' => 'Trường :attribute và :other phải giống nhau.',
+    'size' => [
+        'numeric' => 'Trường :attribute phải bằng :size.',
+        'file' => 'Dung lượng tập tin trong trường :attribute phải bằng :size kB.',
+        'string' => 'Trường :attribute phải chứa :size kí tự.',
+        'array' => 'Trường :attribute phải chứa :size phần tử.',
+    ],
+    'starts_with' => 'Trường :attribute phải được bắt đầu bằng một trong những giá trị sau: :values',
+    'string' => 'Trường :attribute phải là một chuỗi kí tự.',
+    'timezone' => 'Trường :attribute phải là một múi giờ hợp lệ.',
+    'unique' => 'Trường :attribute đã có trong cơ sở dữ liệu.',
+    'uploaded' => 'Trường :attribute tải lên thất bại.',
+    'url' => 'Trường :attribute không giống với định dạng một URL.',
+    'uuid' => 'Trường :attribute phải là một chuỗi UUID hợp lệ.',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify custom validation messages for attributes using the
+    | convention "attribute.rule" to name the lines. This makes it quick to
+    | specify a specific custom language line for a given attribute rule.
+    |
+    */
+
+    'custom' => [
+        'attribute-name' => [
+            'rule-name' => 'custom-message',
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Validation Attributes
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used to swap our attribute placeholder
+    | with something more reader friendly such as "E-Mail Address" instead
+    | of "email". This simply helps us make our message more expressive.
+    |
+    */
+
+    'attributes' => [
+        'address'               => 'địa chỉ',
```

Comment:
```suggestion
        'address' => 'địa chỉ',
```
Lùi hết về 1 dấu cách thui nha